app/src/main/java/com/example/pokedexdesafio/domain/model/PokemonDetail.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.pokedexdesafio.domain.model
+
+data class PokemonDetail(
+    val id: Int?,
+    val name: String?,
+    val types: List<TypeList>,
```

Comment:
Types can be a list of string

Reply:
Changed to Type, and it contains a String